node/sc/api_bridge.go

```diff
@@ -361,6 +442,21 @@ func (sb *SubBridgeAPI) RegisterToken(cBridgeAddr, pBridgeAddr, cTokenAddr, pTok
 	return nil
 }
 
+func (sb *SubBridgeAPI) RegisterToken(cBridgeAddrStrP, pBridgeAddrStrP, cTokenAddrStrP, pTokenAddrStrP *string) error {
```

Comment:
It seems that `cBridgeAddrStrP` can be either a child bridge address or an alias. But, the name of the parameter confused me. How about changing the name to `cBridgeAddrOrAlias, `firstParam`, or a more intuitive name. 

`parseBridgeAddrWithAlias` also looks similar.

Reply:
Considering this API call:
```
subbridge_registerToken(ALIAS, 0XCHILD_TOKEN, 0XPARENT_TOKEN)
or
subbridge_registerToken(0XCHILD_BRIDGE, 0XPARENT_BRIDGE, 0XCHILD_TOKEN, 0XPARENT_TOKEN)
```
The first and second params are perceived as the following two sets:
1. alias and child token address
2. child bridge address and parent bridge address

Your suggestion hits for the first param, but not the second param in the case above. `FirstParam`, `SecondParam` does not have any identifiable info at all too. Considering the current namings, developers can figure out all the convertible cases by referring to the number of params from `console/web3ext/web3ext.go`.

We may consider changing all the names verbosely, like those representing possibly convertible cases (e.g., cBridgeAddrOrAlias, pBridgeOrChildToken). Please feel free to add further comments.

Reply:
I prefer the verbose style names than inaccurate name.

Reply:
Renamed.